pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1226,6 +1231,27 @@ void validateTableTenantConfig(TableConfig tableConfig) {
     }
   }
 
+  void validateTimeColumnConfig(TableConfig tableConfig, Schema schema) {
```

Comment:
is this the right place? can we create a tableconfigvalidator @icefury71 you were also adding some validations right. lets move all validations related to tableconfig to one class